x-pack/test/functional/es_archives/dashboard/feature_controls/security/data.json

```diff
@@ -175,7 +175,7 @@
   "value": {
     "index": ".kibana",
     "type": "doc",
-    "id": "query:okjpgs",
+    "id": "query:OKJpgs",
```

Comment:
I was observing a failure locally that I was not seeing in CI, and the cause appears to have been case-sensitive saved object IDs. The test that was failing was the one that performed the following steps:

1. Load the saved object by title, where the title is `OKJpgs`, with the ID `query:okjpgs`
2. Change the query
3. Save again, but with the new id `query:OKJpgs`
4. You now have two objects with duplicate titles

This was only possible in the functional tests where we had assigned the IDs manually.